app/src/main/java/com/sdpteam/connectout/profile/ProfileRepository.java

```diff
@@ -7,8 +7,10 @@ public interface ProfileRepository {
 
     /**
      * saves the given Profile in the repository
+     *
+     * @return a boolean indicating if the operation is successful
      */
-    void saveProfile(Profile profile);
+    CompletableFuture<Boolean> saveProfile(Profile profile);
```

Comment:
That's nice! I hope we will have a feedback (instead of void) on every other communication with backend